### ISSUE TITLE
linecode switches

### ISSUE THREAD

**Issue by sanderclaeys** [sanderclaeys/pmd-data-model#17]
Should switches have a linecode?

**Comment by ccoffrin** [comment]
Can you remind me the situation when switches have losses?

**Comment by pseudocubic** [comment]
original fr_bus -> ideal switch -> virtual bus -> virtual branch -> original to_bus

**Comment by ccoffrin** [comment]
Let me rephrase, when do one want to model non-ideal switches?  As I recall, @frederikgeth was promoting a use case, but it might have been only to match the semantics of switches in OpenDSS.

**Comment by ccoffrin** [comment]
I can imagine an argument for switches to have a resistance and reactance.  But conductance on the from and to side seems very odd to me.  I would recomend justifying that or removing it.

**Comment by pseudocubic** [comment]
That’s fair, I don’t think I’ve seen a real example of nonzero conductance/reactance on a switch

**Comment by pseudocubic** [comment]
switches will still support `rs`, `xs`, and `linecode`, but not support `b` an `g` and throw warnings if a linecode is used where these values are significant
